src/main/java/by/itech/lab/supplier/repository/ItemInApplicationRepository.java

```diff
@@ -0,0 +1,23 @@
+package by.itech.lab.supplier.repository;
+
+import by.itech.lab.supplier.domain.ItemsInApplication;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+import java.util.List;
+import java.util.Set;
+
+public interface ItemInApplicationRepository extends JpaRepository<ItemsInApplication, Long> {
```

Comment:
Please, rename the class name. ApplicationItemRepository for example. ItemsInApplication begins to spread wrong name to other classes.